Akalya- doordash home test

akalyamohan opened the pull request:
All test cases have been completed

hbahuguna2 commented:
some methods are missing teardown. It will be better to create setup and tearDown methods in a BaseClass and then extend test classes from the BaseClass to avoid code duplication and missing teardown in some cases. Please fix these issues in your next commit.

akalyamohan commented:
I have added BaseTest class with @BeforeClass for chrome setup and @AfterClass for browser close.
I have removed @AfterMethod in each test class and I have removed chrome set up in @BeforeMethod but kept URL invoking code only in each test class.
File download test alone does not extends BaseTest class as it has different chrome config for file downloading.
Completed and saved the changes as mentioned.
